### ISSUE TITLE
[ig/webai] Web&AI Interest Group

### ISSUE THREAD

**Issue by ruoxiran** [w3c/strategy#511]
# Evaluation

cc: @dontcallmedom 
The rapid growth of AI technologies both inside and outside W3C increasingly impact the Web. Currently, there is no single, official place within W3C to coordinate broad AI-related discussions. Existing groups are either narrowly focused (like WebML WG) or fragmented across overlapping Community Groups. To address this, we propose establishing a Web & AI Interest Group to serve as a formal entry point for AI discussions—tracking trends, connecting groups, supporting workshops(e.g., [web and AI Agent workshop](https://github.com/w3c/strategy/issues/507)), and exploring how AI agents protocols may influence the Web.

* Draft Charter

[Draft Charter](https://w3c.github.io/charter-drafts/2025/webai-ig.html)

Expected end of charter refinement phase: End of August 

[Diff with charter template](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fwebai-ig.html)

* Will this work help to lead the web to its full potential?

Yes. As I mentioned in the first paragraph, AI becomes embedded in Web applications, browsers, and other web infrastructure, there is a growing need for coordination across standardization efforts. 

* Is the work Rec-track ready?

No, this is an IG, Deliverables will be AI-related reports.
  
* Do we have the ecosystem of participants needed to make the work successful?

Yes. The group could draw from a diverse ecosystem across AI and Web stakeholders:

1. Participants could come from W3C Members, including browser vendors, cloud providers, telecoms, academic institutions, and AI companies, and users (e.g., Google, Microsoft, ByteDance, Huawei, China Mobile)

2. Interest: Early support could come from participants in the W3C team who are interested in AI stuff, WebML WG, Webagent CG, AI Agent Protocol CG, etc

3. Related existing W3C web tech: WebNN, WASM, WebGPU, RDF/JSON-LD, AI APIs in WebML CG, DID, VC, etc

The group is also planning to coordinate with external orgs such as the IETF and ITU-T.

digging deeper:

Will we be able to make it succeed?
* right participants interested. What does the ecosystem look like?
W3C Members from browser vendors, cloud providers, telecoms, academic institutions, AI companies, and users.
* implementation likely
No implementation expected.
* manageably sized problem
Need to find Chairs to support this work.
* achievable timeline
2-year recharter
* minimum viable product
Outputs: reports, workshops, and coordination outcomes
* does it raise architectural issues that need to be addressed before it can succeed?
Not yet.

Special considerations?
* risk factors
The main challenge is sustained engagement and maintaining relevance amid rapid AI evolution
* incentives
The growing influence of AI on the Web presents an opportunity to explore its long-term impact, identify future standardization paths, and help shape the AI-related Web technologies.
* openness, decentralization
 All deliverables will be public; group discussions will be conducted on GitHub and public mailing lists.

## Procedural

* [Advance notice to AC](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JulSep/0022.html) when team feel a WG charter is likely to be developed, we send "advance notice" to the AC and public-new-work.
* Horizontal review: in this phase, team should request review of charter-in-development from a11y, i18n, arctec, privsec.

**Comment by TzviyaSiegman** [comment]
Sustainability is not a horizontal (yet), but would it mitigate some potential concerns to call out Sustainable Web IG as a coordinating group?

**Comment by plehegar** [comment]
[Advance notice](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JulSep/0022.html)

**Comment by ruoxiran** [comment]
Based on the feedback from everyone, I’ve opened an [issue](https://github.com/w3c/exploration-ig/issues/8) in the Exploration IG to discuss how we might collaborate on AI-related work.

**Comment by xfq** [comment]
A few suggestions:

> "Working Group or the associated Community Group" – Should be "Interest Group".

> "Working Drafts" should be "Draft Notes"?

Should external coordination include more (e.g., ISO/IEC JTC 1/SC 42 for AI standards)?

Use consistent ampersand or "and" ("Web & AI Interest Group" vs. "Web and AI Interest Group").

"Machine: Learning" should be "Machine Learning".

**Comment by himorin** [comment]
No comment or request from i18n

(sorry for late, dropped from my task list [after period expired...](https://lists.w3.org/Archives/Team/w3t-archive/2025Sep/0043.html))

**Comment by simoneonofri** [comment]
From Security, we would like to ask you to work togheter as joint deliverables as Threat Models related to AI:
 - [AI In Browser](https://github.com/w3c-cg/threat-modeling/blob/main/models/ai-in-browser.md)
 - [Agentic AI Web Browsert](https://github.com/w3c-cg/threat-modeling/blob/main/models/agentic-ai-web-browsers.md)

And adding SING as a liaison also to reflect the Security aspects of AI.

**Comment by ruoxiran** [comment]
> Charter looks good in generl.
> 
> > Note: This draft charter is currently in the Evaluation stage and does not yet have a homepage, github, mailing list, etc. The missing information of the draft charter will be added after approval.
> 
> OK, but If this is being readied for AC review, some of the @[@todo](https://github.com/todo)@@ should be completed. In particular the chair names and affiliations are critical to be added once they are decided, and before AC review starts.

Thanks for reivew,
There is one candidate still waiting for confirmation from his manager. I will add them when he confirms.

> 
> s/`[name] (Working|Interest)`/Web & AI
> 
> ` pick one, or both, as appropriate` suggest picking GitHub issues as a better way to track discussions. Experience shows that email subject lines do not work well here in longer discussions.

updated at: https://github.com/w3c/charter-drafts/pull/712/files
